examples/elasticsearch_outlet.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import random
+import time
+from datetime import timedelta
+
+import elasticsearch
+from databay import Inlet, Link, Outlet
+from databay.inlets import RandomIntInlet
+from databay.planners import ApsPlanner
+from databay.record import Record
+
+TEXT = """
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Phasellus ex erat, viverra tincidunt tempus eget, hendrerit sed ligula. 
+Quisque mollis nibh in imperdiet porttitor. Nulla bibendum lacus et est lobortis porta.
+Nulla sed ligula at odio volutpat consectetur. Sed quis augue ac magna porta imperdiet interdum eu velit. 
+Integer pretium ultrices urna, id viverra mauris ultrices ut. Etiam aliquet tellus porta nisl eleifend, non hendrerit nisl sodales. 
+Aliquam eget porttitor enim. 
+"""
+
+
+class DummyTextInlet(Inlet):
+
+    def __init__(self, text: list, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.text = text
+        self._id = 0
+
+    def pull(self, update):
+        text_selection = random.choice(self.text)
+        self._id += 1
+        time.sleep(1)
+        return {self._id: text_selection}
+
+
+class ElasticSearchIndexerOutlet(Outlet):
+    " An example outlet for indexing text documents from any `Inlet`."
```

Comment:
Doesn't this need three quotation marks `"""`? Just checking, thought it was necessary.

Reply:
I think three is only needed for multiline. Since this was a oneliner I've left it as standard double quotes.

Reply:
Didn't know that! 👍